.azure-pipelines/impacted_area_testing/calculate-instance-numbers.yml

```diff
@@ -8,7 +8,6 @@ parameters:
     default: ""
 
   - name: PREPARE_TIME
-    type: number
```

Comment:
Why is this removed?

Reply:
For cases where the type-specification is `number`. For `PREPARE_TIME` and `NUM_ASICS` it has been removed because this allows template expansion to work. The variables $(NUM_ASIC) will be accepted as string at template expansion step and will later be evaluated to a real value at runtime.

Reply:
I'm still wondering the necessity of removing this.

Reply:
Because of the way Azure pipeline evaluates the pipeline. Template expansions happen at compile time. During this time it does not have the value for `$(NUM_ASIC)` which is evaluated at run-time. Setting a type-restriction will cause the evaluation to fail complaining that `$(NUM_ASIC)` is not a number. Removing it will accept the variable literally and then set to proper value at run-time.